### DISCUSSION TITLE
Walls with different materials front and back

### DISCUSSION THREAD

**Discussion by grrobic** [firemodels/fds#12720] [Q&A] [answered]
Hello,

I would like to clarify if my understanding of asymmetrically layered surfaces is correct. 

User guide in 8.3.1 states: 
If BACKING=’EXPOSED’, the last MATL_ID is applied to the opposite face of the solid. If in the example above, BRICK WALL was applied to the entire OBST using SURF_ID, then when doing a heat transfer calculation from the +x face to the -x face, FDS would consider the OBST to be BRICK followed by INSULATOR and the same for a heat transfer calculation from the -x face to the +x face. To avoid this, specify a second SURF that has the reverse MATL_ID and use SURF_ID6 to apply the two SURF definitions to opposite faces of the OBST.

User guide in 8.3.5 states:
On the -x side of the OBST, layer 1 is MATERIAL A, layer 2 is MATERIAL B, and layer 3 is MATERIAL C. On the +x side, the SURF is applied in the same manner, layer 1 is MATERIAL A, layer 2 is MATERIAL B, and layer 3 is MATERIAL C. This means that both sides of the OBST will compute heat transfer assuming MATERIAL A is the first layer but this is not possible.

Is layer 1 the first layer that is exposed to gas phase and successive layers are not directly exposed to gas phase? If I want to model fire in a room which has walls made of plasterboard (layer 1), brick (layer 2) and insulation (layer 3), then plasterboard (layer 1) is the inner side of the wall that is exposed to gas phase and insulation (layer 3) is the outer side of the wall? And if BACKING=’EXPOSED’ then insulation (layer 3) is exposed to ambient conditions (whatever is set on the other side of the wall)? Or is it neccessary to define second SURF with reversed layers and apply that to the outside surface of the wall?

Thank you.

**Comment by mcgratta** [comment, marked as answer]
You must define a second SURF with reversed layer order.

**Reply by grrobic** [reply]
Just to make it sure, that means surfaces inside the room are given "SURF" and surfaces outside the room are given "reversed SURF"?

**Reply by drjfloyd** [reply]
Yes.  

However, you only need to worry about this if the outside surface of the wall is inside your domain (i.e. you have gas cells next to the outside surface) and you care about what happens in the gas outside the compartment.

**Reply by grrobic** [reply]
I would like to have the outside surface of the wall exposed to outside (gas) environment with BACKING = EXPOSED or VOID to include heat transfer through outer walls. Generally what happens outside is not so much of interest, it is just to include heat loss through the wall.
